bin/crew

```diff
@@ -1605,28 +1500,25 @@ def remove(pkgName)
           puts "Removing #{line} will break crew. It was #{'NOT'.lightred} deleted." if @opt_verbose
         else
           puts "Removing file #{line}".lightred if @opt_verbose
-          begin
-            File.unlink line.chomp
-          rescue StandardError => e # swallow exception
-          end
+          FileUtils.rm_rf line
         end
       end
 
       # remove all directories installed by the package
       File.foreach("meta/#{pkgName}.directorylist", chomp: true) do |line|
-        puts "Removing directory #{line}#{''.lightred}" if @opt_verbose
-        Dir.rmdir line
-      rescue StandardError => e # swallow exception
+        next unless Dir.empty?(line)
+
+        puts "Removing directory #{line}".lightred if @opt_verbose
+        FileUtils.rmdir(line)
```

Comment:
Do we need rmdir_f ?

Reply:
`undefined method 'rmdir_f' for FileUtils:Module`😅

Reply:
But then we need the rescue exception, no?

Reply:
Why do we need to hide the error? (the only case (in normal case) that will cause an error is a non-empty directory, which is handled on line 1509)

Reply:
`Fileutils.rmtree` instead of `FileUtils.rmdir` would give the `rm -rf` functionality which doesn't need to be rescued...

https://ruby-doc.org/stdlib-2.7.0/libdoc/fileutils/rdoc/FileUtils.html#method-c-rmdir

Reply:
I replaced the rmdir with rmtree.